### ISSUE TITLE
Can't upload

### ISSUE THREAD

**Issue by DamienCassou** [crowdin/crowdin-cli#526]
**Describe the bug**

In issue #523, @andrii-bodnar said that v3.9.2 would fix uploading. Nevertheless, I can still reproduce the issue.

**To Reproduce**

```bash
$ npm exec --package @crowdin/cli -- crowdin upload sources --branch=foo --config crowdin.yml
npm WARN exec The following package was not found and will be installed: @crowdin/cli@3.9.2
Downloading java runtime environment for version 11
C:\npm\cache\_npx\e624e2fddcf46226\node_modules\node-fetch\lib\index.js:1519
			destroyStream(response.body, err);
			                       ^

TypeError: Cannot read properties of null (reading 'body')
    at C:\npm\cache\_npx\e624e2fddcf46226\node_modules\node-fetch\lib\index.js:1519:27
    at IncomingMessage.<anonymous> (C:\npm\cache\_npx\e624e2fddcf46226\node_modules\node-fetch\lib\index.js:1749:6)
    at Object.onceWrapper (node:events:627:28)
    at IncomingMessage.emit (node:events:513:28)
    at emitCloseNT (node:internal/streams/destroy:138:10)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
```

As you can see in the second line, @crowdin/cli@3.9.2 is being installed.

**Expected behavior**

I would expect `upload` to work.

**Environment:**
 - OS: Windows
 - Version windows-latest from Azure pipelines

**Comment by andrii-bodnar** [comment]
@DamienCassou, just released a new version - [3.9.3](https://github.com/crowdin/crowdin-cli/releases/tag/3.9.3), could you please try in now?

**Comment by DamienCassou** [comment]
This works brilliantly! Great job guys. Thank you.
